user_management/src/user/views.py

```diff
@@ -186,3 +187,18 @@ def post(request):
             return JsonResponse(data={'errors': ['Invalid JSON format in the request body']}, status=400)
         except Exception as e:
             return JsonResponse(data={'errors': [f'An unexpected error occurred : {e}']}, status=500)
+
+
+@method_decorator(csrf_exempt, name='dispatch')
+class UserIdView(View):
+    @staticmethod
+    def get(request, user_id):
+        if not user_id.isnumeric():
+            return JsonResponse(data={'errors': ['User id must be an integer']}, status=400)
```

Comment:
You can remove this if you change <str:user_id> to <int:user_id> in urls.py

Reply:
Yes I know, as previously said, all of this is on purpose. I want to handle strings (and strings related errors)